EpikCord/client/http.py

```diff
@@ -328,9 +346,28 @@ async def log_request(res, body: Optional[dict] = None):
         finally:
             logger.debug("".join(messages))
 
+    def add_file(
+        self, form: aiohttp.FormData, file: File, i: int
+    ) -> SendingAttachmentData:
+        form.add_field(
+            f"files[{i}]",
```

Comment:
a format like `file_x` would seems more natural?

Reply:
iirc this is how Discord requires it to be, let me check again.